dockergen/nginx.tmpl

```diff
@@ -71,6 +72,7 @@ server {
 {{ range $url, $containers := groupByMulti $containersByHost "Env.VIRTUAL_URL" "," }}
     location {{ $url }} {
         proxy_pass http://{{ $host }}{{ replace $url "/" "-" -1 }}u/;
+        proxy_redirect https://{{ $host }}{{ replace $url "/" "-" -1 }}u/ /;
```

Comment:
Ok, so I have disabled redirect remapping and if I enter http://mydomain.com/ I get a 302 response with Location: https://mydomain.com-u/users/sign_in (redirect to login page that is obviously incorrect).